cudamapper/src/matcher_gpu.cu

```diff
@@ -13,7 +13,6 @@
 #include <cassert>
 #include <numeric>
 
-#include <thrust/scan.h>
```

Comment:
Isn't that needed for `thrust::transform_inclusive_scan`?

Reply:
I believe not, that's why we are including `#include <thrust/transform_scan.h>`
Documentation example also does not include it: https://thrust.github.io/doc/group__transformed__prefixsums_ga88fdc1014d4c4a23995d1730e24ef919.html#ga88fdc1014d4c4a23995d1730e24ef919

Reply:
Ah right, makes sense. Did not see that there.